### PR TITLE
feat: keep track of the last logged in profile

### PR DESCRIPTION
**Pull request by jeeanribeiro**
# Description of change

Keep track of the last logged-in profile using persistent storage of the active profile id

## Links to any relevant issues

Resolves #1562
Closes https://github.com/iotaledger/firefly/issues/2050

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Update (a change that updates existing functionality)

## How the change has been tested

- Select a profile
- Refresh the application
- Enter pin screen is the first loaded one and the select profile is the first step one
- Click on the previous button
- Refresh the application
- Select profile screen is the first loaded one

## Change checklist

Add an `x` to the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that my latest changes pass CI tests
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: packages/shared/routes/login/Login.svelte [hunk 1]
```diff
@@ -49,6 +50,7 @@
         if (prevState) {
             state = prevState
         } else {
+            state = LoginState.Init
```
Comment (begonaalvarezd) [inline packages/shared/routes/login/Login.svelte:53]: why is this line needed here?

**Review by begonaalvarezd — COMMENTED** [review]
Was thinking that it might be a good idea not clearing up the active profile on logout? The app triggers logout in 2 scenarios: 
- User explicitly clicks on logout
- Auto logout after timeout

I think in both scenarios we have also suggest the previous logged profile? 
wdyt?

File: packages/shared/routes/login/Login.svelte (hunk 1 above)
Comment (jeeanribeiro) [reply packages/shared/routes/login/Login.svelte:53]: If you start the app it does not have a previous state, so when the EnterPin screen is active I need a way to return to the main screen with the profile list

File: packages/shared/routes/login/Login.svelte [hunk 2]
```diff
@@ -19,9 +20,19 @@
     let state: LoginState = LoginState.Init
     let stateHistory = []
 
-    const _next = (event) => {
+    onMount(() => {
+        if (get(activeProfileId)) {
```
Comment (begonaalvarezd) [inline packages/shared/routes/login/Login.svelte:24]: Just realized we have to check if the `activeProfileId` exists, people can delete profiles directly removing the folder in their system

Comment (begonaalvarezd) [reply packages/shared/routes/login/Login.svelte]: resolved

**Review by begonaalvarezd — APPROVED** [review]
